src/server/api/routers/club.ts

```diff
@@ -64,6 +64,17 @@ export const clubRouter = createTRPCRouter({
 
     return clubs.slice(0, 5);
   }),
+  byNameNoLimit: publicProcedure
```

Comment:
LMAO, we should probably just rewrite the byName api to accept a limit. don't worry about it on this pr though